api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessagePayloadFactory.java

```diff
@@ -20,6 +20,7 @@
 /**
  * Factory object for building message payloads for emails.
  */
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
We should consider refactoring to create an `EmailManager` (or similar, but not `EmailSender` as it won't actually be sending anything) that builds the message payload and puts messages on the queue. Then the building of email message payloads can be moved out of this class.